x-pack/plugins/security_solution/public/common/components/sourcerer/index.test.tsx

```diff
@@ -70,19 +78,24 @@ describe('Sourcerer component', () => {
     wrapper.find(`[data-test-subj="sourcerer-trigger"]`).first().simulate('click');
     expect(
       wrapper.find(`[data-test-subj="indexPattern-switcher"]`).first().prop('selectedOptions')
-    ).toEqual(mockOptions);
+    ).toEqual([mockOptions[0]]);
   });
-  it('Mounts with some options selected', () => {
+  it('Mounts with multiple options selected', () => {
     const state2 = {
       ...mockGlobalState,
       sourcerer: {
         ...mockGlobalState.sourcerer,
+        kibanaIndexPatterns: [
+          state.sourcerer.defaultIndexPattern,
+          { id: '1234', title: 'auditbeat-*' },
+          { id: '1234', title: 'packetbeat-*' },
```

Comment:
Does this get ignored in the `selectedPatterns` list because `auditbeat-*` has the same id?

Reply:
hmm that should never happen IRL since you cant have the same id in the system, so its not something i wrote for